sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -325,6 +325,18 @@ public partial class SimpleFieldAttribute : System.Attribute
 }
 namespace Azure.Search.Documents.Indexes.Models
 {
+    public partial class AmlSkill : Azure.Search.Documents.Indexes.Models.SearchIndexerSkill
+    {
+        public AmlSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs, string resourceId) { }
+        public AmlSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs, System.Uri scoringUri) { }
+        public AmlSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs, System.Uri scoringUri, string authenticationKey) { }
+        public string AuthenticationKey { get { throw null; } set { } }
+        public int? DegreeOfParallelism { get { throw null; } set { } }
+        public string Region { get { throw null; } set { } }
```

Comment:
Is this an https://apiview.dev/Assemblies/Review/77bdbcef49a3465fa96fac3fb745bf57#Azure.Core.AzureLocation?

Reply:
Actually, do we want to rename the property to `Location` (in just .NET) so it's `public AzureLocation? Location { get; }` rather than `public AzureLocation? Region { get; }`?